src/clock/clock.tsx

```diff
@@ -6,8 +6,9 @@ export type ClockProps = ConnectedProps<typeof connector>;
 
 class Clock extends React.PureComponent<ClockProps> {
   public render() {
-    var digitalMinutes = this.props.clock.minutes;
-    var digitalHours = this.props.clock.hours;
+    const digitalMinutes = this.props.clock.minutes;
+    const digitalHours = this.props.clock.hours;
+    const digitalDays = this.props.clock.days;
```

Comment:
const {
      clock: { minutes, hours, days },
    } = this.props;

then you can just refer to these values as minutes, hours, days